blocks/editable/block.scss

```diff
@@ -0,0 +1,16 @@
+.drop-cap-true .blocks-editable__tinymce:not( :focus )  {
```

Comment:
I was kind of expecting us to only provide block stylesheets in the frontend (block.scss only inside the library folder), but I may be wrong. 

I think we should do:
 - Moving the drop cap style to the `text` block styles because it's an option specific to the text block.
 - `.blocks-editable__tinymce` we should drop this className from the styling because it's an "edit" className only.
 - I don't think we should style a global `drop-cap-true` className but more something like `wp-block-text.is-drop-cap-enabled` (We should probably always enforce the generated block className as a root of these styles shared with the frontend.

Thoughts?